templates/search_page.html

```diff
@@ -84,6 +84,7 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
 
 </div>
 
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
```

Comment:
@sananand007 I figured out how to make the dropdown button works. We missed the reference of source JS code needed by BootStrap 5.0. See its Getting Start page. https://getbootstrap.com/docs/5.0/getting-started/introduction/